src/index.html

```diff
@@ -11,7 +11,195 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" onsubmit="onSubmit()">
+      <div class="block">
+        <fieldset>
+          <legend>Personal information</legend>
+
+          <label for="surname" class="form-field">
+            Surname:
+
+            <input
+            type="text"
+            name="surname"
+            id="surname"
+            required
+            >
+          </label>
+
+          <label for="name" class="form-field">
+            Name:
+
+            <input
+            type="text"
+            name="name"
+            id="name"
+            required
+            >
+          </label>
+
+          <label for="age" class="form-field">
+            How old are You?
+
+            <input
+            type="number"
+            id="age"
+            min="0"
+            value="12"
+            max="100">
```

Comment:
add 2 indents to attributes, fix everywhere
```suggestion
            <input
              type="number"
              id="age"
              min="0"
              value="12"
              max="100"
            >
```